src/hotspot/share/gc/z/zMappedCache.cpp

```diff
@@ -571,10 +666,10 @@ size_t ZMappedCache::remove_from_min(size_t max_size, ZArray<ZVirtualMemory>* ou
 
 void ZMappedCache::print_on(outputStream* st) const {
   // This may be called from error printing where we may not hold the lock, so
-  // values may be inconsistent. As such we read the _entry_count only once. And
+  // values may be inconsistent. As such we read the _tree.size() only once. And
   // use is_empty_error_reporter_safe and size_error_reporter_safe on the size
   // class lists.
-  const size_t entry_count = Atomic::load(&_entry_count);
+  const size_t entry_count = _tree.size();
```

Comment:
There doesn't seem to be an `Atomic::load` or `volatile` to make sure that we honor the comment about reading only once.